src/librados/IoCtxImpl.h

```diff
@@ -206,6 +206,7 @@ struct librados::IoCtxImpl {
   int aio_writesame(const object_t &oid, AioCompletionImpl *c,
 		    const bufferlist& bl, size_t write_len, uint64_t off);
   int aio_remove(const object_t &oid, AioCompletionImpl *c);
+  int aio_remove(const object_t &oid, AioCompletionImpl *c, int flags);
```

Comment:
yeah, I'd rather use aio_operate() from the cli instead of adding a new method to librados.